envoy/upstream/cluster_manager.h

```diff
@@ -36,6 +36,12 @@
 #include "absl/container/node_hash_map.h"
 
 namespace Envoy {
+
+// Forward declaration.
+namespace Server {
+class Instance;
```

Comment:
Why is this needed?

Reply:
Good catch, thanks!
It was needed in a previous iteration of this PR, but not anymore, so I removed it.